iot/manager/manager.js

```diff
@@ -104,7 +104,7 @@ function lookupRegistry (client, registryId, projectId, cloudRegion, cb) {
       console.log(err);
     } else {
       console.log('Looked up existing registry');
-      console.log(data);
+      console.log(data.data);
```

Comment:
Calling this `data.data` is confusing.  The callback argument should be named `res`, or `response`, as it contains an HTTP response object.